src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = days * 40;
+  const minTwenty = 20;
+  const minFifty = 50;
+  const three = 3;
+  const seven = 7;
```

Comment:
```suggestion
  const longTerm = 7;
```
А якщо період для великої знижки зміниться?